easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.3.0-2.27-LAPACK-3.6.1.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
+
+lapackver = '3.6.1'
```

Comment:
LAPACK 3.7.0 is also available, but building OpenBLAS with it doesn't seem to be working, see https://github.com/xianyi/OpenBLAS/issues/1042, so sticking to 3.6.1